grails-app/domain/org/gmod/chado/Cvterm.groovy

```diff
@@ -122,29 +122,29 @@ synonyms                        : Synonym
     static mappedBy = [
 //			arraydesignsForPlatformtypeId: "TODO",
 //	                   arraydesignsForSubstratetypeId: "TODO",
-cvtermRelationshipsForObjectId  : "cvtermByObjectId",
-cvtermRelationshipsForSubjectId : "cvtermBySubjectId",
-cvtermRelationshipsForTypeId    : "cvtermByTypeId",
-cvtermpathsForObjectId          : "cvtermByObjectId",
-cvtermpathsForSubjectId         : "cvtermBySubjectId",
-cvtermpathsForTypeId            : "cvtermByTypeId",
-cvtermpropsForCvtermId          : "cvtermByCvtermId",
-cvtermpropsForTypeId            : "cvtermByTypeId",
-cvtermsynonymsForCvtermId       : "cvtermByCvtermId",
-cvtermsynonymsForTypeId         : "cvtermByTypeId",
-expressionCvtermsForCvtermId    : "cvtermByCvtermId",
-expressionCvtermsForCvtermTypeId: "cvtermByCvtermTypeId",
-phenotypesForAssayId            : "cvtermByAssayId",
-phenotypesForAttrId             : "cvtermByAttrId",
-phenotypesForCvalueId           : "cvtermByCvalueId",
-phenotypesForObservableId       : "cvtermByObservableId"
+cvtermRelationshipsForObjectId  : "object",
+cvtermRelationshipsForSubjectId : "subject",
+cvtermRelationshipsForTypeId    : "type",
+cvtermpathsForObjectId          : "object",
+cvtermpathsForSubjectId         : "subject",
+cvtermpathsForTypeId            : "type",
+cvtermpropsForCvtermId          : "cvterm",
+cvtermpropsForTypeId            : "type",
+cvtermsynonymsForCvtermId       : "cvterm",
+cvtermsynonymsForTypeId         : "type",
+expressionCvtermsForCvtermId    : "cvterm",
+//expressionCvtermsForCvtermTypeId: "cvtermByCvtermTypeId",
+phenotypesForAssayId            : "assay",
+phenotypesForAttrId             : "attr",
+phenotypesForCvalueId           : "cvalue",
+phenotypesForObservableId       : "observable"
```

Comment:
I think that this is probably correct so long as it matches on the other side.  There are no foreign keys on table other than cv_id and dbxref.